css/index.css

```diff
@@ -79,6 +129,22 @@ p {
     border-bottom: 1px dashed black;
 }
 
+.Top-about {
+    display: flex;
+    flex-wrap: wrap;
+    padding-bottom: 20px;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+}
+
+.Top-about p {
+    margin-left: 20px;
+}
+
+.Top-about h2 {
+    margin-left: 20px;
+}
+
 .top-content .text-container {
```

Comment:
while it wont break anything, it is custom to keep css classes and ids lowercased